packages/hardhat-chai-matchers/test/properPrivateKey.ts

```diff
@@ -0,0 +1,45 @@
+import { expect, AssertionError } from "chai";
+
+import "../src";
+
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+
+describe("Proper private", () => {
```

Comment:
Can you please update all these test descriptions (and `it` blocks) to say "private key" instead of just "private"?

Reply:
Fixed in [b07ff26](https://github.com/NomicFoundation/hardhat/pull/2627/commits/b07ff26264bbdec399261bb28f754ada0875bb46).